app-config.yaml

```diff
@@ -67,6 +67,12 @@ proxy:
     headers:
       Cookie: ${ARGOCD_AUTH_TOKEN}
 
+  '/prometheus/api':
+    # url to the api of your hosted argoCD instance
+    target: http://localhost:9090/api/v1/
```

Comment:
is this intended to work with [Thanos](https://thanos.io/tip/components/query.md/) as well?

Reply:
If it exposes the same HTTP API as vanilla Prometheus, same queries should work AFAIK.